tests/MimeTypeTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace GuzzleHttp\Tests\Psr7;
+
+use GuzzleHttp\Psr7;
+
+class MimeTypeTestTest
```

Comment:
Should be `MimeTypeTest` ?

Reply:
The current file in the 1.x branch is also named `MimeTypeTestTest`. But I agree, might as well :)